packages/maskbook/src/web3/hooks/useTransactionState.ts

```diff
@@ -15,6 +15,8 @@ export enum TransactionStateType {
     FAILED,
     /** Reject by external provider */
     REJECTED,
+    /** Workaround: ITO swapp fail due to unlucky*/
+    ITO_UNLUCKY,
```

Comment:
If the unlucky status means the transaction failed, then you should use `TransactionType.FAILED`.

Reply:
It is not FAILED, but Confirm.